userguide/payment/includes/payment-overview.adoc

```diff
@@ -43,11 +43,12 @@ The figure below shows the relationship between a Kill Bill `Account`, its vario
 
 image::https://docs.google.com/drawings/d/1ERbfXS0LKSyANT08wnp3zDyoROkhKWSdX2EK0LpwLQ4/pub?w=960&amp;h=480[align=center]
 
-Kill Bill provides many open-source https://docs.killbill.io/latest/payment_plugin.html#payment-plugin-overview[payment plugins] corresponding to different payment gateways. In addition, Kill Bill also has a https://github.com/killbill/killbill-plugin-api[Plugin API] which can be used for developing a custom payment plugin corresponding to a specific payment gateway as explained in our https://docs.killbill.io/latest/payment_plugin.html[payment plugin development guide].
+Kill Bill provides many open-source https://docs.killbill.io/latest/plugin_introduction.html#payment_gateways[payment plugins] corresponding to different payment gateways. In addition, Kill Bill also has a https://github.com/killbill/killbill-plugin-api[Plugin API] which can be used for developing a custom payment plugin corresponding to a specific payment gateway as explained in the https://docs.killbill.io/latest/payment_plugin.html[_Plugin Development Guide_].
 
-By default, Kill Bill does not come configured with any payment plugins. So, if you would like to install an existing plugin or a custom plugin, you need to do it via the https://github.com/killbill/killbill-cloud/tree/master/kpm[Kill Bill package manager (kpm)].
+By default, Kill Bill does not come configured with any payment plugins. So, if you would like to install an existing plugin or a custom plugin, you need to do this seprately as explained in the https://docs.killbill.io/latest/plugin_installation.html[_Plugin Installation Guide_].
 
-Kill Bill does however have a built-in payment method called __EXTERNAL_PAYMENT__. This can be used to track payments that occur outside of Kill Bill. For example, a customer may be invoiced through Kill Bill, and may later make a payment by check. The payment needs to be recorded into the system to mark the invoice as being paid and to bring its balance to zero. In such cases, the payment is recorded against the __EXTERNAL_PAYMENT__ payment method.
+
+Kill Bill does however have a built-in payment method called __EXTERNAL_PAYMENT__. This can be used to track payments that occur outside of Kill Bill. For example, a customer may be invoiced through Kill Bill, and may later make a payment by check. The payment needs to be recorded into the system to mark the invoice as being paid and to bring its balance to zero. In such cases, the payment can be recorded against the __EXTERNAL_PAYMENT__ payment method.
```

Comment:
```suggestion
However, Kill Bill does have a built-in payment method called __EXTERNAL_PAYMENT__. This can be used to track payments that occur outside of Kill Bill. For example, a customer may be invoiced through Kill Bill and may later make a payment by check. The payment needs to be recorded into the system to mark the invoice as being paid and to bring its balance to zero. In such cases, the payment can be recorded against the __EXTERNAL_PAYMENT__ payment method.
```
Are we putting `__EXTERNAL_PAYMENT__` in code font?

Reply:
Yes, let's use code font. Also, the name of the payment method is `__EXTERNAL_PAYMENT__` (It has leading and trailing underscore characters). Asciidoc converts this to Italics, so I've added a `\` character to prevent this. Let me know how this looks.

Reply:
Use two backslash characters instead of just one. (Otherwise, it gets italicized.)

"To prevent two adjacent characters (e.g., __, ##), from being interpreted as AsciiDoc syntax you need to precede it with two backslashes (\\)."
https://docs.asciidoctor.org/asciidoc/latest/subs/prevent/